cross-modal-search/README.md

```diff
@@ -70,128 +58,146 @@ for encoding image and text respectively.
+   
+![](visualizations/query-flow.png)
+This Flow shows what happens when a user queries our data. First, the provided text description is passed through the text Encoder which turns
+it into a vector. Now, we use our image vector index to find the most similar image encodings to the previously computed text vector from the user.
+Because the user does not want to see the vector as a result, but the image this vector belongs to we use the key-value lookup to get from image vector to human-interpretable JPG image.
+Note, that this Flow only shows how to search for images using text. The example actually support searching for text using images as well. 
+As an exercise, you can think of the required steps for that and check against our [Flow configuration](flows/flow-query.yml).
+
+## 🔮 Overview of the files
+
+*Add a list with all folders/files in the example:*
+
+|                      |                                                                                                                  |
+| -------------------- | ---------------------------------------------------------------------------------------------------------------- |
+| 📂 `flows/`          | Folder to store Flow configuration                                                                               |
+| --- 📃 `flow-index.yml`     | YAML file to configure indexing Flow                                                                             |
+| --- 📃 `flow-query.yml`     | YAML file to configure querying Flow                                                                             |
+| 📂 `pods/`           | Folder to store Pod configuration                                                                                |
+| --- 📂 `clip`   | YAML files to configure the Encoder using the clip model                                                                               |
+| --- 📂 `vse`   | YAML files to configure the Encoder using the vse model                                                                                |
+| --- 📃 `image-load.yml`   | YAML file to configure loading JPG files                                                                             |
+| --- 📃 `image-normalzer.yml`   | YAML file to configure how images are normalized                                                                           |
+| --- 📃 `index-image-kv.yml`   | YAML file to configure the key-value image index                                                                           |
+| --- 📃 `index-image-vector.yml`   | YAML file to configure the vector image index                                                                           |
+| --- 📃 `index-text.yml`   | YAML file to configure the index for the text vectors                                                                            |
+| --- 📃 `merge_matche_sort_topk.yml`   | YAML file to configure the ranking of matches                                                                        |
```

Comment:
Is this file being used? I don't see it in the Flow diagram

Reply:
It is used by the query flow right after the `image_kv_indexer` but does not appear in the visualization (I think because it is part of the pods) 
```yaml
  - name: image_kv_indexer
    polling: all
    uses: pods/index-image-kv.yml
    shards: $JINA_SHARDS
    uses_after: pods/merge_matches_sort_topk.yml
    remove_uses_ba: true
```